### ISSUE TITLE
An attempt was made to reference a token that does not exist

### ISSUE THREAD

**Issue by leary1337** [wailsapp/wails#2506]
### Description

When compiling on Windows 10 without admin privileges, the implementation fails ("ERR | CreateWebResourceResponse Error: An attempt was made to reference a token that does not exist.") and then the application opens.

Console:
```
DEB | Using go webview2loader
ERR | CreateWebResourceResponse Error: An attempt was made to reference a token that does not exist.
```

![telegram-cloud-photo-size-2-5240491616387581517-y](https://user-images.githubusercontent.com/59512774/226199231-f41825e8-61fc-4620-9e91-8814d74682fa.jpg)

GOARCH=amd64
GOOS=windows
CGO_ENABLED=1

### To Reproduce

1. wails init
2. wails build -windowsconsole

### Expected behaviour

Runs immediately and without errors.

### System Details

```shell
# System

OS           | Windows 10 Pro
Version      | 2009 (Build: 19045)
ID           | 22H2
Go Version   | go1.20.2
Platform     | windows
Architecture | amd64

# Wails

Version | v2.4.0

# Dependencies

Dependency | Package Name | Status    | Version
WebView2   | N/A          | Installed | 111.0.1661.41
npm        | N/A          | Installed | 9.5.0
*upx       | N/A          | Available |
*nsis      | N/A          | Available |
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
```

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

This issue has already been fixed on master with PR #2496.

**Comment by leaanthony** [comment]
This is now fixed in v4.2.1 👍
